.azure/pipelines/stress.yml

```diff
@@ -0,0 +1,38 @@
+# This configuration builds the repository and runs stress
+
+# Don't run CI for this config
+trigger:
+  batch: true
```

Comment:
So my presumption here is that this is basically a rolling build. Whenever there are new commits and isn't a running build, it will build the current master.

Sounds good to start with.

Reply:
Yeah. This is currently timed to run every 3 hours on the master branch in AzDO. I just changed it to fail the run if there are any failures. No log parsing yet.